docs/code/reference.md

```diff
@@ -125,6 +125,7 @@ than good.
 
 ## Variable names, class names, function names {#variable-names-class-names-function-names}
 
+Variable names should be lower case while Class names should be camel cased. Contants should be written in all capitals.
```

Comment:
I don't think this is relevant for this PR?

Reply:
Honestly its not but I wasn't sure how much love this page gets, its a little neglected, so I added some clarification to a few spots I thought needed it.

Reply:
Please put this in a separate PR. It helps to focusing discussions, it creates more focuses commit messages etc.